### DISCUSSION TITLE
Hide/show the application window.

### DISCUSSION THREAD

**Discussion by netbit** [emilk/egui#588] [Q&A]
Is it possible to set the visibility of the main application window?

**Comment by coderedart** [comment]
It depends on the backed. Winit has set_visible function for its window.

**Reply by netbit** [reply]
How do I do this using egui_glium?

**Reply by coderedart** [reply]
unfortunately, that option is not exposed in the egui_glium/epi itself. you need to get the windowedContext from glutin and call the [https://docs.rs/glutin/0.27.0/glutin/window/struct.Window.html#method.set_visible](set_visible) method on it.
